tests/test_hub_mixin_pytorch.py

```diff
@@ -85,6 +96,66 @@ def test_from_pretrained_model_id_only(self, from_pretrained_mock: Mock) -> None
         from_pretrained_mock.assert_called_once()
         self.assertIs(model, from_pretrained_mock.return_value)
 
+    def pretend_file_download(self, **kwargs):
+        DummyModel().save_pretrained(self.cache_dir, config=TOKEN)
+        return self.cache_dir / "model.safetensors"
+
+    @patch.object(DummyModel, "_hf_hub_download")
```

Comment:
```suggestion
    @patch("huggingface_hub.hf_hub_download")
```

Mocking like this should work and avoid the alias

Reply:
Ha, didn't get this to work with the existing imports. My lack of python mock experience is exposed ;) Switching to `import huggingface_hub` makes it work. If you can enlighten me if there is a way to use `@patch` with a fqn while using `from .file_download import hf_hub_download`, more than happy to update the PR.

Reply:
Today I learned - thanks for taking care of that